crates/bevy_core_pipeline/src/mipmap_generator.rs

```diff
@@ -0,0 +1,620 @@
+//! A wrapper that runs a shader repeatedly to populate a series of mipmaps.
+//!
+//! A variety of passes need to or will need to do this, and using this module
```

Comment:
A one sentence explanation with a link of what mipmaps are will make this a lot more appraochable.